plenum/test/client/test_faulty_client_with_faulty_node.py

```diff
@@ -28,6 +28,7 @@ def nodeChangesRequest(txnPoolNodeSet):
 
 
 # noinspection PyIncorrectDocstring,PyUnusedLocal,PyShadowingNames
+@pytest.mark.skip(reason='sdk integration')
```

Comment:
It should be discussed

Reply:
Rewrite the test
do not change node modifications
just send req and check resp

Reply:
Rewritten